ansible/inventory/env/group_vars/all.yml

```diff
@@ -197,6 +197,7 @@ sunbird_dial_repo_api_key: "{{ core_vault_sunbird_ekstep_api_key }}"
 sunbird_plugin_repo_api_base_url: "http://{{searchservice_ip}}:9000" 
 sunbird_data_service_api_base_url: "{{sunbird_ekstep_api_base_url}}"
 sunbird_data_service_api_key: "{{ core_vault_sunbird_ekstep_api_key }}"
+sunbird_content_service_api_base_url: "http://{{sunbird_swarm_manager_lb_ip}}:9002"
```

Comment:
is it correct @G33tha

Reply:
@rjshrjndrn  its correct, we have to override this variable in private repo common.yml to point it to private ingress loadbalancer ip.
@PradyumnaNagendra please update this variable in private repo also, 
 http://{{private_ingressgateway_ip}}/<service>

Reply:
:+1:

Reply:
ok